<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RDP Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/res/logo.png</normaloff>:/res/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_connect">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Connect(&amp;C)</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/res/refresh.png</normaloff>:/res/refresh.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+C</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>HOSTNAME</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/res/host.png</normaloff>:/res/host.png</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IP ADDRESS</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/res/ip.png</normaloff>:/res/ip.png</iconset>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_monitor">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Monitor(&amp;M)</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/res/connect.png</normaloff>:/res/connect.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+M</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_addr">
           <property name="text">
            <string>Server Address</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_addr">
           <property name="text">
            <string>192.168.6.130</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_port">
           <property name="text">
            <string>Server Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="text">
            <string>8080</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_file">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/res/browser.png</normaloff>:/res/browser.png</iconset>
       </attribute>
       <attribute name="title">
        <string>File Transfer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0" colspan="6">
         <widget class="QTreeWidget" name="treeWidget_local">
          <property name="cursor" stdset="0">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Attr</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QTreeWidget" name="treeWidget_remote">
          <property name="cursor" stdset="0">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Attr</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>108</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <widget class="QToolButton" name="toolButton_upload">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/res/upload.png</normaloff>:/res/upload.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QToolButton" name="toolButton_download">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/res/download.png</normaloff>:/res/download.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_screen">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/res/monitor.png</normaloff>:/res/monitor.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Screen Monitor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_monitor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>monitorSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_upload</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>uploadSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_download</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>downloadSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_local</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>localSlot(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_remote</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>remoteSlot(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>pushButton_monitor</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemEntered(QTableWidgetItem*)</signal>
   <receiver>pushButton_monitor</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>monitorSlot()</slot>
  <slot>connectSlot()</slot>
  <slot>uploadSlot()</slot>
  <slot>downloadSlot()</slot>
  <slot>localSlot(QTreeWidgetItem*,int)</slot>
  <slot>remoteSlot(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
